### ISSUE TITLE
qgis.mx website not available 

### ISSUE THREAD

**Issue by DelazJ** [qgismexico/qgismexico.github.io#1]
Hi,
Just to let you know that https://qgis.mx (listed in https://qgis.org/en/site/forusers/usergroups.html) is not available.

PS: could there be a readme added to this repo to let people know that they are at the right place? Thanks

**Comment by rnanclares** [comment]
Thanks for the heads @DelazJ , a mexican company, which is coincidentally a qgis bronze sponsor, stole our domain, because we had a problem with the credit card that was configured for the automatic renovation. We've asked them repeatedly to give the domain back but they haven't  complied.

**Comment by DelazJ** [comment]
Indeed, while looking around for the working URL the other day, i came across that https://qgis.com.mx and wanted to report the infringement to psc but seems that it is on their radar already. That's a shame and a behavior incompatible with the QGIS spirit. Hope you get this fixed as soon as possible.

**Comment by rnanclares** [comment]
Yes, I talked with Marco Bernasocchi to warn the PSC in both cases. The company still doesn't see this is a grave violation of the QGIS copyright and local domain rules. What can i say... it's just a pity.
